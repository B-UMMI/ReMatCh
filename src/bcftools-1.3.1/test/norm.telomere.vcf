##fileformat=VCFv4.2
##contig=<ID=22,length=30>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
22	17	.	NN	N	.	.	.
