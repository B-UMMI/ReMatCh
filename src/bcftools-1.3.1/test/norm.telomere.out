##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##contig=<ID=22,length=30>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
22	1	.	NN	N	.	.	.
